runtime/autoload/health/provider.vim

```diff
@@ -312,7 +312,10 @@ function! s:check_python() abort
 
   " No Python executable could `import neovim`, or host_prog_var was used.
   if !empty(pythonx_errors)
-    call health#report_error('Python provider error:', pythonx_errors)
+    call health#report_warn(pythonx_errors, ["See :help provider-python for more information.", 
+          \ "You may disable this provider (and warning) by adding `let g:loaded_python3_provider = 0` to your init.vim"
+          \])
+"If you wish to disable the Python provider (and this warning), then "
```

Comment:
this a stray string that's turned into a comment?

Reply:
can be removed as part of the typofix PR ;)